includes/controls/groups/text-stroke.php

```diff
@@ -0,0 +1,121 @@
+<?php
+namespace Elementor;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Elementor text stroke control.
+ *
+ * A base control for creating text stroke control. Displays input fields to define
+ * the text stroke and color stroke.
+ *
+ * @since 3.5.0
+ */
+class Group_Control_Text_Stroke extends Group_Control_Base {
+
+	/**
+	 * Fields.
+	 *
+	 * Holds all the text stroke control fields.
+	 *
+	 * @since 3.5.0
+	 * @access protected
+	 * @static
+	 *
+	 * @var array Box shadow control fields.
+	 */
+	protected static $fields;
+
+	/**
+	 * Get text stroke control type.
+	 *
+	 * Retrieve the control type, in this case `box-shadow`.
+	 *
+	 * @since 3.5.0
+	 * @access public
+	 * @static
+	 *
+	 * @return string Control type.
+	 */
+	public static function get_type() {
+		return 'text-stroke';
+	}
+
+	/**
+	 * Init fields.
+	 *
+	 * Initialize text stroke control fields.
+	 *
+	 * @since 3.5.0
+	 * @access protected
+	 *
+	 * @return array Control fields.
+	 */
+	protected function init_fields() {
+		$controls = [];
+
+		$controls['text_stroke'] = [
+			'label' => esc_html__( 'Text Stroke', 'elementor' ),
+			'type' => Controls_Manager::SLIDER,
+			'size_units' => [ 'px', 'em', 'rem' ],
+			'range' => [
+				'px' => [
+					'min' => 0,
+					'max' => 10,
+				],
+				'em' => [
+					'min' => 0,
+					'max' => 1,
+					'step' => 0.01,
+				],
+				'rem' => [
+					'min' => 0,
+					'max' => 1,
+					'step' => 0.01,
+				],
+			],
+			'selector' => '{{WRAPPER}}',
+			'selectors' => [
+				'{{SELECTOR}}' => '-webkit-text-stroke: {{SIZE}}{{UNIT}}; stroke-width: {{SIZE}}{{UNIT}}',
+			],
```

Comment:
Is this necessary?
Does
```suggestion
			'selectors' => [
				'{{WRAPPER}}' => '-webkit-text-stroke: {{SIZE}}{{UNIT}}; stroke-width: {{SIZE}}{{UNIT}}',
			],
```
Not work?

Reply:
I added default for 'selector' because if the developer needs to change the selector when implementing this control, it will be easier to change only the 'selector' and not the whole CSS properties with 'selectors.'